examples/multicsc/plot_sample_evoked_response.py

```diff
@@ -8,7 +8,7 @@
 heartbeat and eyeblink artifacts, three atoms of evoked responses, and a
 non-sinusoidal oscillation.
 
-.. [1] Dupré La Tour, T., Moreau, T., Jas, M., & Gramfort, A. (2018).
+.. [1] Dupre La Tour, T., Moreau, T., Jas, M., & Gramfort, A. (2018).
```

Comment:
please don't cripple the name of Tom. It's already a pain for him to get his name right in citations :)

If it's does not work with sphinx-gallery which I doubt it should be fixed there

Reply:
You can add `# -*- coding: utf-8 -*-` at the top if necessary.